tests/wikidata/test_api.py

```diff
@@ -27,3 +28,11 @@ def test_build_composite_description():
 
     assert build_composite_description('kissa', ['kotikissa', 'kesykissa'], 'cat', 'an animal') \
         == 'kissa (kotikissa, kesykissa) [= cat]: an animal'
+
+
+def test_yield_external_results():
+    searched_word = StrInLanguage('eurooppalainen lyhytkarva', 'fi')
+    results = list(yield_external_results(searched_word))
+
+    assert len(results) == 1
+    assert results[0].id == 'Q20793'
```

Comment:
I would extract the `Q20793` to some constant variable

Reply:
I put the value in a constant and tried to give it a better name, although I'm not sure what to call it since it is not very important what the page is about.